projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -152,6 +152,9 @@ function populate_newsletter_access_rows( $column_id, $post_id ) {
 		case Token_Subscription_Service::POST_ACCESS_LEVEL_SUBSCRIBERS:
 			echo esc_html__( 'Subscribers', 'jetpack' );
```

Comment:
Is `esc_html__()`  actually needed here, instead of a simple `__()`

Reply:
Jetpack commit linter doesn't allow this. See the error below

```
FILE: ...lugins/jetpack/extensions/blocks/subscriptions/subscriptions.php
----------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------
 150 | ERROR | All output should be run through an escaping function
     |       | (see the Security sections in the WordPress Developer
     |       | Handbooks), found '__'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
 153 | ERROR | All output should be run through an escaping function
     |       | (see the Security sections in the WordPress Developer
     |       | Handbooks), found '__'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
 156 | ERROR | All output should be run through an escaping function
     |       | (see the Security sections in the WordPress Developer
     |       | Handbooks), found '__'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
----------------------------------------------------------------------
```